### PR TITLE
Add a pydicom recipe.

### PR DESCRIPTION
**Pull request by arokem**
This is related to #284

### CODE REVIEW

File: recipes/pydicom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package:
+  name: pydicom
+  version: "0.9.9"
```
Comment (pelson) [inline recipes/pydicom/meta.yaml]: @jakirkham made a comment about pulling this out into a single variable (https://github.com/conda-forge/staged-recipes/pull/284/files#r58764815). That is maybe something to consider next time you update this recipe.

**Comment by pelson** [comment]
Lovely recipe. :+1:

**Comment by jakirkham** [comment]
Oops, looks like I'm a bit late to the party here. :wink:

That being said, nothing is really final here as we can change it in the feedstock. :smile:

Just noticing that this `pydicom` is using `setuptools`. This will need these arguments ( `--single-version-externally-managed --record record.txt` ) for the `setup.py install` commands. Otherwise looks good. Thanks @arokem.

As a side note, if you know other people interested in these packages, feel free to ping them and ask them their thoughts. Also, please ask if they are interested in being maintainers. That way you can get some help in managing these dependencies as well. Not to mention, this will help make sure that packages added here remain useful for a variety of different use cases.

**Comment by arokem** [comment]
Thanks for the comments and suggestions. What's the correct way to make
changes in the feedstock? PR against the pydicom-feedstock repo?

Once I wrap my head around this, I will report this back to the
neuroimaging in python community, so that we can discuss how we go about
maintaining this as a community. Thanks again!

**Comment by jakirkham** [comment]
> Thanks for the comments and suggestions. What's the correct way to make changes in the feedstock? PR against the pydicom-feedstock repo?

Absolutely. Glad to help. Yes, PR is the best way. That way you can verify everything still passes without having changed `master` yet.

> Once I wrap my head around this, I will report this back to the neuroimaging in python community, so that we can discuss how we go about maintaining this as a community. Thanks again!

Sounds good. Feel free to ask questions if you need. Welcome to conda-forge. :smile:

**Pull request merged**